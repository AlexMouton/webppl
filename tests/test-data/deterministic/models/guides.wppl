[
  scalarParam(1),
  T.sumreduce(tensorParam([3, 2], 1))
];
