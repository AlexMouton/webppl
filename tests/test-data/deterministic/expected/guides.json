{
  "result": [
    1,
    6
  ]
}
